sdk/python/kfp/compiler/testdata/v2_compatible_two_step_pipeline.yaml

```diff
@@ -128,7 +128,7 @@ spec:
         pipelines.kubeflow.org/v2_component: "true"
     initContainers:
     - command: [/bin/mount_launcher.sh]
-      image: gcr.io/ml-pipeline/kfp-launcher:1.6.2
+      image: gcr.io/ml-pipeline/kfp-launcher:dummy
```

Comment:
Why do we need to update this if the test ignores everything after `:` ?

Reply:
It's not required, but changing this can immediately verify that we're ignoring the label. Otherwise, if the change had a bug, we would only discover that the next time we bump the version. 

Other very minor benefits include: 
- when I read `dummy`, I would think this value doesn't matter for test purpose (not tied to a specific release).
- If someone search `1.6.2` across the codebase to update all occurrences or to verify that all occurrences have been updated. Then this won't show un in the search result.

Reply:
Makes sense, thanks!